gen/banned_facility_users

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use Perun::Agent;
+use JSON::XS;
+
+our $SERVICE_NAME = "banned_facility_users";
+our $PROTOCOL_VERSION = "3.0.0";
+my $SCRIPT_VERSION = "3.0.0";
+
+perunServicesInit::init;
+my $DIRECTORY = perunServicesInit::getDirectory;
+my $data = perunServicesInit::getHashedHierarchicalData;
+
+my $agent = Perun::Agent->new();
+my $bansAgent = $agent->getBanOnFacilityAgent;
+
+#Constants
+our $A_U_F_IS_BANNED;		*A_U_F_IS_BANNED = \'urn:perun:user_facility:attribute-def:virt:isBanned';
+
+my %bansStruc;
+my $facilityId = perunServicesInit::getFacility->getId;
+my @bans = $bansAgent->getBansForFacility(facilityId => $facilityId);
+
+foreach my $memberId ($data->getMemberIdsForFacility) {
+	my $isBanned = $data->getUserFacilityAttributeValue( member => $memberId, attrName => $A_U_F_IS_BANNED );
+	if ($isBanned) {
+		my $uid = $data->getUserIdForMember(member => $memberId);
+		foreach my $ban (@bans) {
+			if ($ban->getUserId() eq $uid) {
```

Comment:
`getBan(PerunSession sess, int userId, int faclityId)` exists in FacilityManager, maybe we could add it to bansAgent and then use it here?

Reply:
I 'd say iterating all facility bans will be better because we decrease the number of rpc calls.

Reply:
I see, you’re right.